### PR TITLE
[otbn/doc] Updated secure wipe specification.

### PR DESCRIPTION
**Pull request by vrozic**
Removed user-initiated secure wipe of internal state.
This resolves issue #7468 .

Signed-off-by: Vladimir Rozic <vrozic@lowrisc.org>

### CODE REVIEW

File: hw/ip/otbn/doc/_index.md [hunk 1]
```diff
@@ -687,7 +687,7 @@ Detected integrity violations in the data memory raise a fatal `imem_error`.
 
 ### Secure Wipe {#design-details-secure-wipe}
 
-The entire OTBN state, including the contents of instruction and data memories, can be securely deleted on demand from a host software.
+The contents of instruction and data memories, can be securely deleted on demand from a host software.
```
Comment (rswarbrick) [inline hw/ip/otbn/doc/_index.md:690]: Nit: The comma needs deleting too here.

File: hw/ip/otbn/doc/_index.md [hunk 2]
```diff
@@ -699,12 +699,12 @@ In the following situations OTBN itself initiates a full secure wipe:
 
 The internal state secure wipe is automatically triggered when an OTBN operation completes, either successfully, or unsuccessfully due to a recoverable error.
 
-Host software can trigger a full state wipe by writing `3'b111` to {{< regref "SEC_WIPE">}}  (i.e. set all individual state wipe bits).
+Host software can trigger a data or instruction memory secure wipe by writing `3'b11` to {{< regref "SEC_WIPE">}}  (i.e. by setting both individual state wipe bits to 1).
```
Comment (rswarbrick) [inline hw/ip/otbn/doc/_index.md:702]: This should be "data **and** instruction memory", I think?

File: hw/ip/otbn/doc/_index.md [hunk 3]
```diff
@@ -707,7 +707,7 @@ Detected integrity violations in the data memory raise a fatal `imem_error`.
 
 ### Secure Wipe {#design-details-secure-wipe}
 
-The entire OTBN state, including the contents of instruction and data memories, can be securely deleted on demand from a host software.
+The contents of instruction and data memories can be securely deleted on demand from a host software.
 In addition, full or partial secure wipe is triggered automatically by the OTBN in certain situations.
```
Comment (imphil) [inline hw/ip/otbn/doc/_index.md:711]: That sentence is now missing the internal wipe. Maybe reword the two sentences completely to first describe what types of secure wipe we have, and then how it can be triggered (roughly, it's the intro!)

File: hw/ip/otbn/doc/_index.md [hunk 4]
```diff
@@ -721,12 +721,12 @@ In the following situations OTBN itself initiates a full secure wipe:
 
 The internal state secure wipe is automatically triggered when an OTBN operation completes, either successfully, or unsuccessfully due to a recoverable error.
 
-Host software can trigger a full state wipe by writing `3'b111` to {{< regref "SEC_WIPE">}}  (i.e. set all individual state wipe bits).
+Host software can trigger a data and instruction memory secure wipe by writing `3'b11` to {{< regref "SEC_WIPE">}} (i.e. by setting both individual state wipe bits to 1).
```
Comment (imphil) [inline hw/ip/otbn/doc/_index.md:724]:
```suggestion
Host software can trigger a data and instruction memory secure wipe by writing `2'b11` to {{< regref "SEC_WIPE">}} (i.e. by setting both individual state wipe bits to 1).
```

**Review by imphil — COMMENTED** [review]
Thanks @vrozic! 

Can you also update the commit message to 
- Update the summary line to give more detail what is going on, e.g. "Remove internal state trigger from host"
- Indicate *why* this change was made.
- Use a format like "Fixes #1234" to auto-close the issue with the PR. (https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

**Review by imphil — APPROVED** [review]

**Comment by imphil** [comment]
LGTM, thanks @vrozic.
